packages/playground/website/src/lib/state/redux/slice-ui.ts

```diff
@@ -0,0 +1,84 @@
+import { createSlice, PayloadAction, Middleware } from '@reduxjs/toolkit';
+
+export type SiteError =
+	| 'directory-handle-not-found-in-indexeddb'
+	| 'directory-handle-permission-denied'
+	| 'directory-handle-directory-does-not-exist'
```

Comment:
@adamziel I don't see either of these errors being created in Playground yet. Should these be removed, or are they errors we expect to report in subsequent PRs?